docs/ja/setup-guide/oss-kubernetes-clusters.md

```diff
@@ -0,0 +1,356 @@
+# OSS Kubernetesクラスタのログの可視化（Loki）
+
```

Comment:
We would better to have a link to Japanese/English on both documents for visitors navigated here with direct link to be aware of the existence of translated document.

Reply:
Like this 
![image](https://github.com/user-attachments/assets/cf4229e0-89d4-481c-a93c-1b4381f0e58a)